### ISSUE TITLE
Syncing stuck when indexing addresses 

### ISSUE THREAD

**Issue by proletesseract** [navcoin/navcoin-core#404]
navcoin.conf
```
rpcuser=[REDACTED]
rpcpassword=[REDACTED]
rpcallowip=127.0.0.1
server=1
txindex=1
addressindex=1
daemon=1
```

debug.log
```
2019-01-24 17:13:29 UpdateTip: new best=d78f16f1a4833bca15c169f49bfbde3570cd75ce56397ea2f658517fcca8386b height=2703251 version=0x7101c1ff log2_work=72.760653 tx=5683967 date='2018-12-15 09:52:32' progress=0.852465 cache=182.7MiB(893296tx)
2019-01-24 17:13:29 UpdateTip: new best=ddc3e2a8632b9b13dd64d3d923465e0440ef7d4beb9c949beebc520f41b1aac4 height=2703252 version=0x710141f7 log2_work=72.760654 tx=5683969 date='2018-12-15 09:53:04' progress=0.852466 cache=182.7MiB(893298tx)
2019-01-24 17:13:42 ERROR: AcceptBlockHeader: block b0775d0528aa65daf5f7bc81b2d37b461c2a805bc654c7b1f2f298a82a0b9177 is marked invalid
2019-01-24 17:13:42 ERROR: invalid header received
2019-01-24 17:13:42 ProcessMessages(headers, 164003 bytes) FAILED peer=1
2019-01-24 17:14:02 ERROR: AcceptBlockHeader: prev block invalid
2019-01-24 17:14:02 Misbehaving: 89.73.51.132:44440 (0 -> 100) BAN THRESHOLD EXCEEDED
2019-01-24 17:14:02 ERROR: invalid header received
```

Block being rejected:

https://www.navexplorer.com/block/b0775d0528aa65daf5f7bc81b2d37b461c2a805bc654c7b1f2f298a82a0b9177

**Comment by craigmacgregor** [comment]
Reconsidering block 2703253 moves the tip forward past this point.
```
reconsiderblock b0775d0528aa65daf5f7bc81b2d37b461c2a805bc654c7b1f2f298a82a0b9177
```

**Comment by aguycalled** [comment]
can you find in the log a previous entry specifying why `b0775d0528aa65daf5f7bc81b2d37b461c2a805bc654c7b1f2f298a82a0b9177` was rejected at first?

**Comment by mxaddict** [comment]
I think this was fixed in 4.7.3 as well, closing for now, just reopen if you experience this on never versions.
